### ISSUE TITLE
No contact

### ISSUE THREAD

**Issue by JunxingHu** [xiexh20/CHORE#2]
Hello,

Thank you for sharing your great work! I am very interested in your work and try to run the demo. However, I found that the contact cannot be detected. The log and results are as follows. Is this a parameter setting problem or something else? Thank you.

[chore_log.txt](https://github.com/xiexh20/CHORE/files/9367760/chore_log.txt)
![k1 rend_overlap](https://user-images.githubusercontent.com/23099566/185297809-c8a8ad12-dc90-4d92-8207-ee9c24dcd0a1.jpg)
![k1 rend_side](https://user-images.githubusercontent.com/23099566/185297819-d84f608b-3c96-4199-baef-214c557225b7.jpg)

**Comment by xiexh20** [comment]
Hi,

Thanks for your interest!
I tried the demo several times, but I've never seen this before :),
have you tried rerun the demo?

**Comment by JunxingHu** [comment]
Thank you for your prompt response! I tried the demo on two servers and several samples from the BEHAVE dataset as below, but none of the blue areas representing contacts were reconstructed. I only modified --obj_name in demo.py for different samples. The inputs include k1.color.jpg, k1.color.json, k1.mocap, k1.mocap.ply, k1.obj_mask.jpg, and k1.person_mask.jpg. Do I need to set other items? Thank you for your kind help.
![chore-behave samples](https://user-images.githubusercontent.com/23099566/185525954-63d21bb1-3059-4bff-83fc-36a32daed7cf.jpg)

**Comment by xiexh20** [comment]
Hi, I think your command is correct, as most of the examples you show are reconstructed reasonably. 
what do you mean *blue areas representing contacts*? Do you refer to the contact sphere shown in the teaser figure? It was added for better visualization, you can refer to [here](https://github.com/xiexh20/behave-dataset/blob/64eb4350e6c8ecb699f772ce031b5cd67a017836/viz/pyt3d_wrapper.py#L118) to check how it is generated.

**Comment by JunxingHu** [comment]
Thank you for your help! I will try adding the contact sphere. However, the reconstructed human body does not touch the basketball as shown in the first question. Is there any other version of the pre-trained model? In addition, since I use the RTX3090, the version of some algorithm packages such as cuda cannot be completely consistent with the requirements. Have you tested it in other environments such as cuda11.0? Thank you very much.

**Comment by xiexh20** [comment]
Hi, I tried this pre-trained model on cuda11.3, Nvidia RTX 8000 and V100. I don't have models trained on other machines. But I tried to run this several times and always got the same result. 
Maybe you can try to rerun this example again and see if you get different results. Or if you can reproduce the numeric results on the BEHAVE test set, it should be fine.

**Comment by Young2647** [comment]
I tried the basketball example and get the same result as @JunxingHu. No contact on the basketball
![k1 rend_side](https://user-images.githubusercontent.com/55054380/186353983-2bf70aaa-a1ef-43f0-87be-adf75f652156.jpg)
.

**Comment by JunxingHu** [comment]
Hello, thank you for your patient help. Since most GPUs do not support environments lower than cuda11.0, could you please provide the versions of all packages in anaconda when cuda11.3 to help us reinstall the environment? Moreover, testing 90 sequences is very time-consuming, so I ran one sequence `BEHAVE/sequences/Date03_Sub03_backpack_back/` by using `python recon/recon_fit_behave.py chore-release --save_name chore-release -s [path to one sequence]` and get the results by using `python recon/evaluate.py` as below. Are these results the same as yours? Thank you.
```python
{'smpl': {'mean': 0.06429596999150605, 'std': 0.021513825524556374}, 'obj': {'mean': 0.07156142227877363, 'std': 0.03903869274395644}, 'total': 41}
{'smpl': {'mean': 0.06429596999150605, 'std': 0.021513825524556374}, 'obj': {'mean': 0.07156142227877363, 'std': 0.03903869274395644}, 'total': 41}
```

**Comment by JunxingHu** [comment]
Wow, thanks for your prompt reply. This is a great work!

**Comment by luckcodingdog** [comment]
I also tried the basketball example and get the same result as @JunxingHu & @Young2647 . No contact on the basketball.
I run the demo on 3090 GPU with pytorch 1.11 and CUDA 11.2.
At the same time, i do the same evaluation with JunxingHu, my result is similar with yours.
